sormas-backend/src/main/java/de/symeda/sormas/backend/util/DtoHelper.java

```diff
@@ -145,6 +145,7 @@ public static void fillDto(EntityDto dto, AbstractDomainObject entity) {
 		return target;
 	}
 
+	// todo this really should return void. Taking a target/receiver argument and then return it anyways does not make sense
```

Comment:
Will this be refactored with a different ticket?

Reply:
tracked in #6880